docs/tools/dynamic-simulation.md

```diff
@@ -0,0 +1,96 @@
+---
+title: dynamic-simulation
+layout: default
+---
+
+The `dynamic-simulation` command is used to run dynamic simulations on the network.
+
+# Usage
+```shell
+$> itools dynamic-simulation --help
+usage: itools [OPTIONS] dynamic-simulation --case-file <FILE> [--help] [-I <property=value>]
+              [--import-parameters <IMPORT_PARAMETERS>] [--output-file <FILE>]
+              [--parameters-file <FILE>] [--skip-postproc]
```

Comment:
the list of parameters and their description should be updated based on the specifications document.